sql/core/pom.xml

```diff
@@ -35,6 +35,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.rocksdb</groupId>
+      <artifactId>rocksdbjni</artifactId>
+      <version>6.2.2</version>
```

Comment:
Shall we use the latest version?
- https://mvnrepository.com/artifact/org.rocksdb/rocksdbjni/6.20.3

Reply:
@dongjoon-hyun Thanks for taking care. Following the comments from Yikun in http://apache-spark-developers-list.1001551.n3.nabble.com/DISCUSS-Add-RocksDB-StateStore-tt30645.html, I planned to upgrade the rocksdb version after all the code shipped. At least >=6.4.6 for the Arm64 support.
Personally, I think it's good to keep the 6.2.2 since this is well tested in the production environment and it's safe to use it in the first stage. I believe the latest RocksDB versions are stable enough, but the state store implementation has its own design on the snapshot/checkpoint file architecture, it's good to spend more time on testing whether we can safely upgrade it to the latest RocksDB version. WDYT?

Reply:
Sounds good to have well tested version first.